pybossa/view/account.py

```diff
@@ -1151,6 +1151,61 @@ def add_metadata(name):
     flash("Input saved successfully", "info")
     return redirect(url_for('account.profile', name=name))
 
+
+@blueprint.route('/<user_name>/taskbrowse_bookmarks/<short_name>', methods=['GET', 'POST'])
+@login_required
+def taskbrowse_bookmarks(user_name, short_name):
+    user = user_repo.get_by_name(name=user_name)
+    if user is None:
+        return abort(404)
+    if current_user.name != user_name:
+        return abort(403)
+
+    all_bookmarks = user.info.get('taskbrowse_bookmarks', {})
+    proj_bookmarks = all_bookmarks.get(short_name, {})
+    # get all bookmarks for project
+    if request.method == 'GET':
+        return jsonify(proj_bookmarks)
+    # add a bookmark
+    if request.method == 'POST':
+        url = request.body.get('url', None)
+        name = request.body.get('name', None)
+        if name is None or len(name) > 100 or \
+            url is None or len(url) > 500:
```

Comment:
For readability, please add variables/macro with suitable names for the numbers being used. ex

Reply:
I chose 100 characters to be max length for name, and 500 characters for the url somewhat arbitrarily. I didn't want for users to be able to store an unbounded string. Do you think 100 and 500 are reasonable sizes respectively?

Reply:
> I chose 100 characters to be max length for name, and 500 characters for the url somewhat arbitrarily. I didn't want for users to be able to store an unbounded string. Do you think 100 and 500 are reasonable sizes respectively?

Yes, bounding strings makes sense. 100 & 500 looks reasonable. Thanks!